rslib/src/decks/schema11.rs

```diff
@@ -116,6 +116,10 @@ pub struct NormalDeckSchema11 {
     extend_new: i32,
     #[serde(default, deserialize_with = "default_on_invalid")]
     extend_rev: i32,
+    #[serde(default, deserialize_with = "default_on_invalid")]
+    review_limit: Option<u32>,
+    #[serde(default, deserialize_with = "default_on_invalid")]
+    new_limit: Option<u32>,
```

Comment:
I presume this was a conscious decision to exclude the daily limits from the schema11 representation? That seems reasonable*, I just wanted to double check it was intentional.

*My comment elsewhere about preserving default for today may influence this though.